tests/Integration/CheckCommandTest.php

```diff
@@ -75,3 +75,39 @@
 
     expect(trim($output))->toContain('INFO  Configuration file already exists.');
 });
+
+it('may fail with text option', function (): void {
+    $application = new Application;
+
+    $application->add(new CheckCommand);
+
+    $command = $application->find('check');
+
+    $commandTester = new CommandTester($command);
+
+    $commandTester->execute([
+        '--text' => 'This is a test with a typoo.',
+    ]);
+
+    $output = $commandTester->getDisplay();
+
+    expect(trim($output))->toContain('Did you mean: typo, typos, type, topi');
```

Comment:
Here we can also test for the correct status code:

```suggestion
    expect(trim($output))->toContain('Did you mean: typo, typos, type, topi')
        ->and($commandTester->getStatusCode())->toBe(1);
```